.github/workflows/ci-test-mac.yml

```diff
@@ -105,5 +105,5 @@ jobs:
       - name: Run tests (MacOS)
         run: |
           source ~/.bash_profile
-          ctest --output-on-failure --schedule-random -E "bpf|ubpf|testgen|smith"
+          ctest --output-on-failure --schedule-random -E "bpf|ubpf|testgen|smith|p4tc|P4TC"
```

Comment:
The filters on the actions are inconsistent.

Reply:
Ohh, missed that one
Nice catch
Fixed it